src/client/sandbox/index.js

```diff
@@ -160,7 +160,7 @@ export default class Sandbox extends SandboxBase {
 
         this.event.initDocumentListening();
 
-        if (isWebKit)
+        if (window.top === window.self && isWebKit)
```

Comment:
https://github.com/DevExpress/testcafe/blob/5cfc571037c21d470e7d5da489974a16cdfd1439/src/client/core/utils/dom.js#L427
according to comment we need to swap `window.top === window.self` and `isWebKit`